pages/signup/register.tsx

```diff
@@ -219,7 +219,7 @@ const Register = () => {
               </div>
               <p className="mb-10 -mt-8 gc-description">
                 {t("signUpRegistration.slaAgreement")}&nbsp;
```

Comment:
Just to avoid any confusion I would rename both `slaAgreement` and `slaAgreementLink` to `termsAgreement` and `termsAgreementLink` in the `register.tsx` and `signup.json` (EN and FR) files.